rest-api/src/main/resources/paths/uploads/v3_uploads_recordId.yml

```diff
@@ -1,10 +1,12 @@
 get:
     operationId: getUploadByRecordId
     summary: Get an upload record by the health data record ID
+    description: A superadmin or worker can retrieve any upload in any app. An Admin can retrieve any upload in their own app. A developer can retrieve an upload generated from their own account in an app.
     tags:
         - Uploads
         - _For Admins
         - _For Workers
+        - _For Developers
```

Comment:
This is an actual code change to the SDK. To make sure we don't have any weird version conflicts, we should probably bump the pom version.

Reply:
Bumped version to 0.21.1.